appveyor.yml

```diff
@@ -1,8 +1,7 @@
 environment:
   global:
       PYTHON: "C:\\conda"
-      CONDA_DEPENDENCIES: "setuptools numpy scipy matplotlib scikit-learn mayavi pandas h5py PIL pyside pytest"
-      PIP_DEPENDENCIES: "git+git://github.com/nipy/PySurfer.git nibabel nitime nose-timer nose-faulthandler pytest-attrib"
```

Comment:
we don't need nitime nose-timer nose-faulthandler anymore?

Reply:
we don't use `nose` anymore